src/workerd/api/basics.c++

```diff
@@ -623,8 +645,9 @@ jsg::Ref<AbortSignal> AbortSignal::any(jsg::Lock& js,
     sig->addEventListener(js, kj::str("abort"), kj::mv(identified),
         AddEventListenerOptions{// Once the abort is triggered, this handler should remove itself.
           .once = true,
-          // When the signal is triggered, we'll use it to cancel the other registered signals.
-          .signal = signal.addRef()});
+          // Each of the followed signals will maintain a strong reference to this new
+          // one that's been created.
+          .followingSignal = signal.addRef()});
```

Comment:
Should we rename signal to `followingSignal` in the entire `any()` method? I feel like it would alleviate some confusion.

Reply:
Possibly I suppose. i don't think that's really worth re-running CI tho.